recipe/bld.bat

```diff
@@ -21,13 +21,17 @@ set BLD_OPTS=%WIN64% ^
     XERCES_DIR=%LIBRARY_PREFIX% ^
     XERCES_INCLUDE="-I%LIBRARY_INC% -I%LIBRARY_INC%\xercesc" ^
     XERCES_LIB=%LIBRARY_LIB%\xerces-c_3.lib ^
-    HDF4_DIR=%LIBRARY_PREFIX% ^
-    HDF4_LIB="%LIBRARY_LIB%\hdf.lib %LIBRARY_LIB%\mfhdf.lib %LIBRARY_LIB%\xdr.lib" ^
```

Comment:
Yes - HDF4 on Windows is only available as a 'pre-VC Features' build - hence built with the wrong compiler for Python 3.x. HDF4 needs to be brought up to date (maybe another conda-forge project?). With the wrong compiler random application crashes will be likely.